pyproject.toml

```diff
@@ -29,8 +29,8 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-typing-extensions = { version = "^4.1.1", python = "<3.8" }
+python = "^3.7"
+typing-extensions = { version = "^4.3.0", python = "<3.11" }
```

Comment:
These changes are due to the desire for `TypedDicts` to be generic. If we exclude that and let the authorization context continue to just be `Dict[str, Any]` then this dependency change can be reverted.